php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -170,7 +170,7 @@ class NDB_BVL_Instrument extends NDB_Page
      */
     static function factory($instrument, $commentID, $page, $guarantee_exists = true)
     {
-        //include_once 'HTML/QuickForm.php';
+        include_once 'HTML/QuickForm.php';
```

Comment:
Travis failed.
Maybe this should remain commented and rely on a comment in the wiki.
Since QuickForm is not in out composer file anymore 
```sudo pear install HTML_QuickForm``` 
should also be provided to users.